plugin.sh

```diff
@@ -106,4 +110,5 @@ fi
     ${CACHE_REPO:-} \
     ${TARGET:-} \
     ${BUILD_ARGS:-} \
-    ${BUILD_ARGS_FROM_ENV:-}
+    ${BUILD_ARGS_FROM_ENV:-} \
+    ${MIRROR:-}
```

Comment:
Newline now missing

Reply:
I'm sorry, I'm very confused, what do you mean "newline now missing"? Do you mean that there should be an empty line on 115?

Reply:
yes ... I need to add the common formation checks ... of our normal repos here :)

I should be able to change it for you :)